app/src/main/java/org/mozilla/tv/firefox/channels/DefaultChannel.kt

```diff
@@ -26,9 +28,20 @@ class DefaultChannel(
         titleView.text = title
     }
 
+    fun setSubtitle(subtitle: CharSequence?) {
+        subtitleView.visibility = if (subtitle.isNullOrEmpty()) {
+            View.GONE
+        } else {
+            View.VISIBLE
+        }
+
+        subtitleView.text = subtitle
```

Comment:
Do we still need to set the text if `subtitle` is empty or null?

Reply:
It seems right to me to clear the text, in case we ever want to update these over time.  It doesn't really matter since the visibility will be GONE anyway, but this also avoids an extra conditional branch.